hubblestack/daemon.py

```diff
@@ -646,6 +647,33 @@ def load_config():
     os.chmod(__opts__['log_file'], 384)
     os.chmod(parsed_args.get('configfile'), 384)
 
+    # Check for a cloned system with existing hubble_uuid
+    cached_uuid_path = os.path.join(os.path.dirname(__opts__['configfile']), 'hubble_cached_uuid')
+    cached_system_uuid_path = os.path.join(os.path.dirname(__opts__['configfile']),
+                                           'hubble_cached_system_uuid')
+
+    try:
+        if os.path.isfile(cached_uuid_path) and os.path.isfile(cached_system_uuid_path):
```

Comment:
Correct. How do you propose we detect that?

Reply:
Would it work if we in addition check for `hubble_cached_uuid` presence and `hubble_cached_system_uuid` lack of presence?
This condition can legitimately come about only when it hasn't ever run long enough for `system_uuid` to be written. The alternative is that a cloned id may slip through.

Reply:
If `hubble_cached_uuid` is present and `hubble_cached_system_uuid`, we have no way of knowing if the system was cloned. In that case, I would argue that we _shouldn't_ delete the `hubble_uuid` file cache. I'd rather it stay the same because the cloned system is the less likely alternative.